### PR TITLE
add brazilian portuguese description

### PR DESCRIPTION
**Pull request by Panemiko**
Translated the en-US.md description file into pt-BR.md with Brazilian Portuguese

### CODE REVIEW

**Comment by Madis0** [comment]
Thanks, I've asked my existing pt_BR translator to review this.

File: Description/pt-BR.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Fabulously Optimized é pensado para funcionar em qualquer mundo ou servidor, [roda em 4 launchers, é hospedado em 3 sites][6] e [suporta diversas linguagens][7].
+
+### Desenvolvimento aberto
+
+Fabulously Optimized [é de código aberto][8] e o mesmo é para maioria de seus mods. É desenvolvido com ideias da comunidade, colaborando com desenvolvedores de mods para criar uma melhor experiência de usuário para todos.
+
+### Construído com o Fabric
+
+Comparado ao Forge, Fabric atualiza e carrega rápido, não inclui bibliotecas gigantes e mantem um grande suporte a mods. [(Sim, Quilt está sendo considerado.)][9]
```
Comment (PaperKing13isPro) [inline Description/pt-BR.md:35]: it should be "mantém", not "mantem".

Comment (Panemiko) [reply Description/pt-BR.md]: fixed

**Comment by Madis0** [comment]
Thank you, now published to CurseForge and Modrinth! If you want a contributor rank in our Discord, tell me your Discord username (with numbers).

**Pull request merged**